tests/apps/binding_output/go.mod

```diff
@@ -2,6 +2,9 @@ module app
 
 go 1.15
 
-require github.com/gorilla/mux v1.7.3
+require (
+	github.com/dapr/go-sdk v1.0.0-rc-3
```

Comment:
i think this might be an issue if there are changes in dapr which is not there in go-sdk causing issues ...